src/smbprotocol/session.py

```diff
@@ -272,7 +271,7 @@ def connect(self):
                 protocol=self.auth_protocol,
             )
         except spnego.exceptions.SpnegoError as err:
-            raise SMBAuthenticationError("Failed to authenticate with server: %s" % str(err.message))
+            raise SMBAuthenticationError("Failed to authenticate with server") from err
```

Comment:
While it's good to do the inner exception I would probably like to keep the original exception error inside the main exception string just for preserving how ti works today or if someone is catching that in their code and doing `str(e)` themselves.

Reply:
Ok